templates/latex/etoc_doxygen.sty

```diff
@@ -1453,26 +1453,26 @@
     \etocmulticolstyle[#1]{#2}%
     \tableofcontents*}
 \newcommand\etoc@local@multicoltoc[2][\etocdefaultnbcol]{%
-    \etocmulticolstyle[#1]{#2}%
-    \localtableofcontents}
+\etocmulticolstyle[#1]{#2}%
+\localtableofcontents}
 \newcommand\etoc@local@multicoltoci[2][\etocdefaultnbcol]{%
-    \etocmulticolstyle[#1]{#2}%
-    \localtableofcontents*}
+\etocmulticolstyle[#1]{#2}%
+\localtableofcontents*}
 \newcommand*\etoc@ruledtoc[2][\etocdefaultnbcol]{%
-    \etocruledstyle[#1]{#2}%
-    \tableofcontents}
+\etocruledstyle[#1]{#2}%
+\tableofcontents}
 \newcommand*\etoc@ruledtoci[2][\etocdefaultnbcol]{%
-    \etocruledstyle[#1]{#2}%
-[O    \tableofcontents*}
+\etocruledstyle[#1]{#2}%
+\tableofcontents*}
```

Comment:
This should indeed be fixed, and probably also be reported upstream (seen the copyright notice), I will check this. The spaces should stay (as they are from the original.

Reply:
I made PR #10171 which incorporates the removal of the stray bytes.  I don't know where these bytes come from as they were not found upstream...